vllm/v1/core/scheduler.py

```diff
@@ -184,6 +197,12 @@ def schedule(self) -> "SchedulerOutput":
             # Schedule the request.
             scheduled_running_reqs.append(request)
             self.scheduled_req_ids.add(request.request_id)
+            if request.use_struct_output:
+                # PERF: in case of chunked prefill,
```

Comment:
@WoosukKwon I updated the notes here, PTAL if this makes sense to you

Reply:
lgtm